docs/headers/ti/getkey.rst

```diff
@@ -10,6 +10,12 @@ ti/getkey.h
 For programs that interact with the OS's user interface, you may want to use processed keystrokes instead of raw scancodes.
 The :code:`os_GetKey` routine handles 2nd and alpha presses, and can even show menus.
 
+TI-83 Premium CE
+----------------
+
+No research has been done on how the TI-83 Premium CE is different here.

```

Comment:
What can I do go help lol

Reply:
I don't think anybody has even looked into GetKey equates for MathPrint stuff, never mind the exact math engine. My *guess* is that all the stuff that's shared with 84PCE is the same and it's just a matter of the extra code for the Exact/Approx toggle key. I don't even know if the menus like Alpha+F1 have their own codes.